vmdb/app/controllers/api_controller.rb

```diff
@@ -95,6 +96,10 @@ def initialize
     @api_token_mgr   = TokenManager.new(@module)
   end
 
+  def self.all_encrypted_attributes
+    (ATTR_TYPES[:encrypted] | ::Vmdb::ConfigurationEncoder::PASSWORD_FIELDS).sort
```

Comment:
yay.

As noted before, there are a few other password fields. but I don't think any of those values should come back